src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -92,6 +110,51 @@ static Box _boundingBox(const Shape* shape)
 }
 
 
+static Box _boundingBox(const Picture* pic)
+{
+    Box bbox;
+    pic->bounds(&bbox.x, &bbox.y, &bbox.w, &bbox.h, true);
+    return bbox;
+}
+
+
+static Box _boundingBox(const Scene* scene)
+{
+    auto& children = const_cast<Scene*>(scene)->paints();
+
+    if (children.empty()) {
+        return {0.0f, 0.0f, 0.0f, 0.0f};
+    }
+
+    Box box = {0.0f, 0.0f, -1.0f, -1.0f}; //init with invalid values
+    Box childBBox;
+
+    for (auto child : children) {
+        childBBox = _boundingBox(child);
+        _mergeBoxes(box, childBBox);
+    }
+
+    return box;
+}
+
+
+static Box _boundingBox(const Paint* paint)
```

Comment:
paint->bounds() will return the merged region of its internal elements. You are recomputing the bounds here. Is this necessary?

Also, It looks  "static Box _boundingBox(const Shape* shape)" can be removed?

Reply:
I need to know the bounds without stroke. for a single shape I can remove it, but for a scene it's not possible, I need to traverse all elements. long time ago I wanted to change the bounds api accordingly, so I could use it here, but as I remember it wasn't approved.

Reply:
hmm but your logic looks including the stroking width as well.

Reply:
ah ok I mis-read the code.